app/models/katello/host/content_facet.rb

```diff
@@ -18,6 +18,8 @@ class ContentFacet < Katello::Model
       validates :host, :presence => true, :allow_blank => false
       validates_with Validators::ContentViewEnvironmentValidator
 
+      attr_accessible :content_view, :content_view_id, :lifecycle_environment, :lifecycle_environment_id, :host
```

Comment:
That's really an abuse of attr_accessible. Moreover, when we drop protected attributes, you won't able to set content_view and lifecycle_environment via mass assignment anyway.